packages/integration-sdk-runtime/src/execution/dependencyGraph.ts

```diff
@@ -281,6 +281,19 @@ export function executeStepDependencyGraph<
                     DisabledStepReason.NONE,
                 );
               skippedStepTracker.add(stepId);
+              // Log child steps disabled
+              const stepDependencies = inputGraph.dependantsOf(stepId);
```

Comment:
So does this handle the scenario where a different step graph is provided? AWS does not always use the default.

Reply:
I confirm you this is taking into account the scenarios where the default graph is not used.  Explanation:
The `inputGraph` that comes as param is created [here](https://github.com/JupiterOne/sdk/blob/main/packages/integration-sdk-runtime/src/execution/step.ts#L86) and the steps that we send to the `buildStepDependencyGraph` function are taking into account the `dependencyGraphId` when it is specified in the integration. That can be seen in [these lines](
https://github.com/JupiterOne/sdk/blob/main/packages/integration-sdk-runtime/src/execution/step.ts#L65-L72)